admin/broadleaf-contentmanagement-module/src/main/java/org/broadleafcommerce/cms/web/controller/BroadleafPageController.java

```diff
@@ -77,7 +77,23 @@ public ModelAndView handleRequest(HttpServletRequest request, HttpServletRespons
         
         // Allow extension managers to override the path.
         ExtensionResultHolder<String> erh = new ExtensionResultHolder<String>();
-        ExtensionResultStatusType extResult = templateOverrideManager.getProxy().getOverrideTemplate(erh, page);
+        Boolean internalValidateFindPreviouslySet = false;
+        ExtensionResultStatusType extResult;
+        
+        try {
+            if (!BroadleafRequestContext.getBroadleafRequestContext().getInternalValidateFind()) {
+                BroadleafRequestContext.getBroadleafRequestContext().setInternalValidateFind(true);
+                internalValidateFindPreviouslySet = true;
+            }
+
+            extResult = templateOverrideManager.getProxy().getOverrideTemplate(erh, page);
+        } finally {
+
+            if (internalValidateFindPreviouslySet) {
+                BroadleafRequestContext.getBroadleafRequestContext().setInternalValidateFind(true);
```

Comment:
again, should it not be false if you previously set it to true in this same scope? Otherwise, it's already true